doc/collections.md

```diff
@@ -1,17 +1,146 @@
 Collections
 ===========
+A collection is a data instance containing a set of (references to)
+instances and relationships between them.
+It is essentially a tripplestore or a local knowledge base that
+semantically describe a domain with some instances and their context.
```

Comment:
A collection is not a triplestore (with one 'p'). Rather, a collection can be described using the RDF framework as a set of triples.